src/index.js

```diff
@@ -1,26 +1,28 @@
 import React, { Suspense, lazy } from 'react';
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import { createRoot } from 'react-dom/client';
-import "./main.css";
+import './main.css';
 
-const App = lazy(() => import('./App.jsx'));
```

Comment:
Hi, Elizabeth. You are removing all code cornering Code Splitting. Please make sure you have the most updated repo before merged